src/editors/object.js

```diff
@@ -758,7 +758,7 @@ export var ObjectEditor = AbstractEditor.extend({
         self.toggleEditJSON()
       })
       this.controls.appendChild(this.editjson_control)
-      this.controls.insertBefore(this.editjson_holder, this.controls.childNodes[1])
+      this.controls.insertBefore(this.editjson_holder, this.controls.childNodes[0])
```

Comment:
What does this change? and why?

I cloned the original repo and tested with ie10 but did not experienced any error

Reply:
When **object editor** create `editjson_holder` to `controls`, the `control` only have 1 child node, but this line try to insert  `editjson_holder` before second child node (which actually not exist).

I test it at ie version of  `10.0.9200.17556`.

Reply:
How is the situation for other browsers?

Reply:
> How is the situation for other browsers?

The exception only ocurr in native ie10(or lower), I test in ie10  mode of ie11 and chrome, old version works all right.